proposals/0009-LRA/0009-LRA.md

```diff
@@ -1459,25 +1459,25 @@ public @interface SRA {
     boolean terminal() default false;
 
     /**
-     * The cancelOnFamily element can be set to indicate which families of HTTP response codes will
-     * cause the SRA to cancel. By default client errors (4xx codes) and server errors (5xx codes)
-     * will result in cancellation of the SRA.
+     * The rollbackOnFamily element can be set to indicate which families of HTTP response codes will
+     * cause the SRA to rollback. By default client errors (4xx codes) and server errors (5xx codes)
+     * will result in rollback of the SRA.
      *
-     * @return the {@link Response.Status.Family} families that will cause cancellation of the SRA
+     * @return the {@link Response.Status.Family} families that will cause rollback of the SRA
      */
     @Nonbinding
-    Response.Status.Family[] cancelOnFamily() default {
+    Response.Status.Family[] rollbackOnFamily() default {
```

Comment:
I think this was likely just carried over from LRA but it seems rollback is more the process than cancel